lib/linguist/languages.yml

```diff
@@ -3332,6 +3332,14 @@ NSIS:
   codemirror_mode: nsis
   codemirror_mime_type: text/x-nsis
   language_id: 242
+Nasl:
```

Comment:
This should be changed to `NASL` (uppercase), as that appears to be the language's canonical spelling (going by its [Wikipedia page](https://en.wikipedia.org/wiki/Nessus_Attack_Scripting_Language)).

Reply:
Good point. Done with new commit!